posts/how-kubernetes-add-to-your-business.md

```diff
@@ -0,0 +1,63 @@
+industry wherein many companies in the insurance to e-commerce to banking industries are bound for generating more 
+focused and specific applications. These applications 
+exist in the form of [microservices](https://blog.ippon.tech/monolithic-to-microservice-consistency-in-distributed-systems/).
+When talking about microservices, its difficult _not_ to talk about containers and container orchestration tools.
+When reading into microservices, you and your developers will ask yourselves: `Would 
+you want to spin up a VM, install an OS, find dependencies, manage security, etc or would you like to spin up a 
+container which does all that for you in seconds?`
+
+Today, we're going to discuss what is Kubernetes and why your business would benefit from this technology. First 
+we'll briefly discuss the differences in microservices and monoliths.
+
+### A Brief History in Microservices
+Before microservices, we had (and still have) monoliths. Applying good development practices, 
+its best to break up your main component into modular pieces. These modular pieces, in the form of microservices, are 
+transforming into containers.
+
+Making monoliths modular into microservices have real cost saving benefits both financially and in developer time:
+- Containers help with keeping deployment times low 
+- The main service operates if a component is down or redeploying
+- Adding new features is independent of the entire monolith 
+- Reduces start-up time for developers
+
+To add to the point, this [research](https://www.researchgate.net/publication/316532483_Cost_comparison_of_running_web_applications_in_the_cloud_using_monolithic_microservice_and_AWS_Lambda_architectures)
+paper from 2015 breaks down the actual cost of using a monolith vs 
+microservice architecture using then-current relevant tech stacks. The researchers found that using a 
+microservice architecture in place of monoliths save up to 70% under similar loads.
+
+Now that I've convinced you that microservices have a place in the world, there are some managerial questions to 
+address: what happens when there are a bunch of microservices running and how does a development team manage them?
+Enter: Kubernetes.
+
+### Kubernetes: A Container Orchestration Solution
+
+Kubernetes began as an internal Google project to manage and reduce costs for Google Search, Youtube, and G-mail. It is 
+a scheduler to most efficiently utilize their clusters. Kubernetes, an open source project, is the result of that 
+inner-source project, Borg. Container orchestration is managing the lifecycle of containers, most notably in larger
+projects or dynamic environments. 
+
+There are many benefits of having a Container Orchestration system such as Kubernetes:
+- provisioning and deploying containers
+- redundancy
+- scalability
+- allocation of resources
+- health monitoring
+
+### Conclusion
+Effectively, Kubernetes is powerful tool to use at your helm where your developers have more time to work on the core
```

Comment:
This says nothing about Ippon, or links the reader to our services.

Reply:
I read some of our other blogs, and I'm not sure if we always have to tie things back to ippon and our services. This was intended to just be an informative bloggy blog. I can change plug ippon in if very much desired

Reply:
My understanding of the audience is that of "non computer people". Correct me if I'm wrong. These people would probably only take away from this article, which brands were mentioned. That's just my take, I'm definitely no expert.

Reply:
hmm alright I'll think of a spin to add to the end, stay tuned 📺

Reply:
Tyler is spot on with his suggestions. Consider the business reader that has heard of Kubernetes but doesn't understand why he/she needs it. Then sell them Ippon as the expert that will implement it for them.

Reply:
Well, the main thing is although we're good at a lot of stuff, we aren't exactly kubernetes experts and therefore can't speak to much k8s experience. With that thought process in mind, my goal was to have something for the business readers to have a supplemental blog to point their technical people to.

Reply:
Think of expert as consulting expert not k8s expert in this context. We don't want to limit sales opportunities based on existing skillsets for hot technologies like k8s. The reason it is a great topic for your blog is the same reason we are looking for these types of projects. We will hire for the right role and we also have resources globally we can tap into. FEED ME K8S PROJECTS :)